### ISSUE TITLE
Copying standard library sources into a resources directory is bad for standard library development.

### ISSUE THREAD

**Issue by dabrahams** [hylo-lang/hylo#932]
You get diagnostics and then you're sent to the copy of the source file, whence you make fixes that don't stick.

- If there's a way to avoid doing that copy, we should do so!
- If we absolutely must do that copy:
    - the files should be made read-only so we don't try to edit them
    - some facility for remapping source locations should be introduced so that diagnostics point to the real source.

**Comment by kyouko-taiga** [comment]
Seems you solved the issue with a PR but I am surprised because it already worked as you expected on my setup.

**Comment by dabrahams** [comment]
@kyouko-taiga I'm confused.  It worked when you build from the command line?  From within Xcode?  Can you double-check your answers by rolling back to an earlier commit and adding a syntax error in the standard library?

**Comment by kyouko-taiga** [comment]
Oh I'm sorry, I got confused because the _tests_ worked as expected (not copied in a bundle), not the standard library. I went back and tested; errors from the stdlib are indeed reported in the bundle. Thanks for your PR.

By the way, I find it simpler to develop new standard library types in a different module before moving them in `Library/Hylo/` once they are complete. That way you don't need to recompile `hc` every time you make a change.
